docs/contributing/pull-requests/index.md

```diff
@@ -97,33 +97,76 @@ changes” to a PR forcing the reviewer to start over again.
-internal ticket is used for prioritization and tracking purposes.</community><bitwarden>Include a
-Jira ticket reference so the reviewer can gain all context on the work.</bitwarden> Tag
-`@dept-design` as a reviewer for any UI changes.
-
-## Review process
+the PR review goes smoothly since it will provide context to the reviewer.
 
 <community>
+Once a community PRs has been created, they will be automatically be linked to an internal Jira ticket. The
+internal ticket is used for prioritization and tracking purposes.
+</community>
 
-Once a Community PR has been created a Bitwarden developer will perform a code review, while we try
-to this in a reasonable time-frame, please understand that we have internal roadmaps and priorities
-that may delay this process.
+<bitwarden>
+When creating the PR, include a Jira ticket reference so the reviewer can gain all context on the work, as well as links to any associated PRs in other repositories.
 
-</community>
+### Tagging reviewers
+
+We use `CODEOWNERS` in each repository to assign the reviewing teams based on the files in the PR.
+These reviews are required for the changes to be merged to `master`.
+
+You can tag additional teams or individuals for review as you see fit, including `@dept-design` for
+any design changes.
+
+### Opening the PR for review
+
+As its name implies, marking a PR as "Ready for Review" indicates that you are ready for all
+assigned teams to review it. If the changes are still in progress, leave the PR in `Draft` status.
+Doing this ensures that reviewers can act on the "Ready for Review" as their signal to begin the
+review process without further notification.
+
+</bitwarden>
+
+## Reviewing the pull request
 
 <bitwarden>
 
-While we mostly use an async review process, please don't hesitate to schedule a meeting with the
-reviewer/contributor to discuss the changes. While async communication can be useful it incurs a
-time penalty which can drag out the review process. And sometimes setting up a short call to discuss
-the changes can potentially save a lot of time.
+At Bitwarden, we believe that the act of reviewing PRs is a critically important part of each
+engineer's job. It is as important, if not more important, than the act of writing code.
+
+To ensure that teams within the organization operate on same set of assumptions for performing
+reviews, we have agreed to a baseline set of expectations.
+
+When a PR author opens a PR for review, they should have the expectation that:
+
+- The act of opening the PR for review is the **only** notification required. Teams are responsible
+  for properly configuring notifications so that team members are aware of their obligations.
+- The reviewing team(s) will respond within **24 hours** to:
+  - Provide a review,
+  - Inform the author when a review will be provided, or
+  - Ask the author to split the work into a smaller PR for review
```

Comment:
Has the team agreed to a 24 hour PR review deadline? I haven't seen any discussion about this, but I may have missed it.

Reply:
Not to my knowledge. I'm not against this though but it should be communicated.

I also think we need to spend further effort to clean up tech-leads codeowners since I haven't found a way to exclude them from review filters.

Reply:
I agree.  I wanted to discuss here and then communicate in a bulleted list to the teams in Slack.  This felt like a good place to start the discussion.  I’m not in any way pushing for 24 - if we want to have a longer timeframe that’s fine.  My main concern is having everyone bought in on what the timeframe is so we don’t have to resort to pinging on Slack for every PR.

Reply:
Maybe "business day"? I personally feel one day is actually pretty tough unless engineers can be in formal support rotations and have this as a priority.

Reply:
I'm not sure what would be a reasonable deadline here, although I agree 1 business day is tight. 2 business days?

This is not a very visible place for discussion and this would be a fairly major change to the current expectations, I suggest sharing in Slack.